README.md

```diff
@@ -82,37 +82,56 @@ These are the empty repositories that have PRs waiting to be merged.
 
 ### rust-vmm
 
-- [vm-device](https://github.com/rust-vmm/vm-device): a virtual machine device
-  model crate.
-- [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
-  implementation for virtio primitives such as virtqueues and descriptor chain.
+- [mshv](https://github.com/rust-vmm/mshv): workspace for Microsoft Hypervisor
+  safe wrappers.
+- [vm-allocator](https://github.com/rust-vmm/vm-allocator): abstractions for
+  allocating resources needed by the VMM while running (such as GSI numbers,
+  MMIO ranges).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio): workspace for virtio
+  devices and related helpers.
 - [vmm-reference](https://github.com/rust-vmm/vmm-reference): reference VMM
   built solely with `rust-vmm` crates and minimal glue.
+- [vhost-device](https://github.com/rust-vmm/vhost-device): A workspace for
+  devices implemented using the
+  [vhost-user-backend](https://github.com/rust-vmm/vhost-user-backend).
```

Comment:
This one has been published to crates.io: https://crates.io/crates/vhost-user-backend

Reply:
The vhost-user-backend is already in the "published" section. Here I was just mentioning it in the context of the vhost-device workspace, which is using it. Can we leave it as is, or do you want me to update this link to be the crates.io one as well?